scripts/unfoldv2.py

```diff
@@ -252,6 +259,11 @@ def getArgsParser(arguments_list=None, print_help=False):
                         help="If True, run unfolding also with IBU for comparison")
     parser.add_argument('-w', '--weight-type', type=str, default='nominal',
                         help="Type of event weights to retrieve from ntuples")
+    parser.add_argument('--scheduler-names', help="Name of the learning rate scheduler to be used", type=str, nargs='+')
+    parser.add_argument('--reduce-on-plateau', type=int, default=0,
+                        help="number of epoch to wait before reducing learning rate")
+    parser.add_argument('--scheduler-args', help="extra arguments for the requested scheduler", type=str)
+    parser.add_argument('--learning-rate', type=float, default=0.001, help="Initial learning rate")
```

Comment:
Would it be possible to group all these arguments into one e.g. `--scheduler-config`, which takes a config file? And the default file would contain the best config that you recommend.